### ISSUE TITLE
Empty table cells are not rendered correctly in text format

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1544]
Using `make text`, a table

```
+-----+-----+
| XXX | XXX |
+-----+-----+
|     | XXX |
+-----+-----+
| XXX |     |
+-----+-----+
|     |     |
+-----+-----+
| XXX | XXX |
+-----+-----+
```

is translated to

```
+-------+-------+
| XXX   | XXX   |
+-------+-------+
+-------+-------+
+-------+-------+
+-------+-------+
| XXX   | XXX   |
+-------+-------+
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1544
- Originally reported by: Kirill Simonov
- Originally created at: 2014-08-25T21:25:37.906

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-26 04:43:47+00:00_

confirmed

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-26 11:52:57+00:00_

`make text` generate wrong table when it has empty table cells. Closes #1544

→ <<cset 438f9fe3de7aeacb2ad396224c07520bece526a4>>
